pallets/crowdloans/src/lib.rs

```diff
@@ -247,33 +263,39 @@ pub mod pallet {
                 Error::<T>::CTokenAlreadyTaken
             );
 
-            if let Some(vault) = Self::current_vault(crowdloan) {
-                if vault.phase != VaultPhase::Failed && vault.phase != VaultPhase::Expired {
-                    return Err(DispatchError::from(Error::<T>::ParaIdAlreadyTaken));
```

Comment:
why this was removed?

Reply:
It was accidentally removed when resolving merge conflicts earlier. Adding back now